content/blog/2024/202402-03-review.rst

```diff
@@ -0,0 +1,112 @@
+==================================================================
+sciwork February and March 2024 review: CFD workshop and scisprint
+==================================================================
+
+:date: 2024-05-12
+:author: Ming-Jie Kuo
+:category: blog
+
+Weekly Meetup
+-------------
+During the meetup events in February and March, modmesh contributors made the installation process
+of modmesh on windows computers smoother and fixed many CI related issues, while some contributors
+developed many plotting related features to make the presentation of the mesh more flexible.
+
+In addition to project development, there were also many people having questions about their work.
+Some people asked about the difficulties they encountered when working on the Vue.js project and
+auto-streaming databases, and some people asked how to improve the performance of LLM models from
+both software and hardware perspectives.
+
+March Workshop
+--------------
+Sciwork organized a new event called Advanced CFD software development workshop in March. Chun-Hsu and
+many sciwork volunteers have been preparing for this workshop since February. Yung-Yu introduced the
+numerical method space-time CESE method, unstructured mesh with mixed elements, and many important
+software engineering techniques to make participants understand how to write the programs to get correct
+results with high-performance and maintainable numerical software through the modmesh project.
+
+Mesh has always been an important issue in numerical software that is easier ignored. While preparing for
+this workshop, modmesh contributors make a lot of effort to finish plotting the unstructured mesh with
+mixed elements. The participants can see a more concrete mesh, which is pretty important for the development
+of numerical software development.
+
+After this workshop many participants discussed on the discord channels to continue to learn about the numerical
+software modmesh. Some participants joined the March scisprint to learn more about the numerical software. sciwork
+will continue to organize more events to enable more people who are interested in numerical software to write
+programs and share ideas together.
+
+February and March Scisprint
+----------------------------
+Two scisprint events were held in February and March, one in Hsinchu in February and one in Taipei in March.
+After the CFD workshop in March, the scisprint in March welcomed many new partners. Everyone started to discuss
+and develop projects of their own interest, and some people came to communicate with each other about their lives and works.
+
+After the CFD workshop in March, modmesh welcomed many new friends. While building modmesh with the new friends, modmesh
```

Comment:
The sentence of *After the ...* seems to duplicate with the previous paragraph.

Reply:
I changed the sentence pattern.